Client/Assets/ContextMenu.js

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+
+"use strict";
+
+var longPressTimeout;
+
+window.addEventListener("touchstart", function (event) {
+  // Don't show the context menu for multi-touch events.
+  if (event.touches.length !== 1) {
+    return;
+  }
+
+  longPressTimeout = setTimeout(function () {
+    var data = {};
+    var element = event.target;
+    do {
+      if (!data.link && element.localName === "a") {
```

Comment:
Do you think we can do some kind of visual effect here? Like do a css animation to show a box with rounded corners around the link element that faded away in like 0.25 seconds? Would that be a good alternative to the native link highlighting that WebKit does?